modules/repository-s3/src/internalClusterTest/java/org/elasticsearch/repositories/s3/S3RepositoryThirdPartyTests.java

```diff
@@ -48,8 +54,23 @@
 import static org.hamcrest.Matchers.lessThanOrEqualTo;
 import static org.hamcrest.Matchers.not;
 
+@ThreadLeakAction({ ThreadLeakAction.Action.WARN, ThreadLeakAction.Action.INTERRUPT })
+@ThreadLeakScope(ThreadLeakScope.Scope.NONE)
 public class S3RepositoryThirdPartyTests extends AbstractThirdPartyRepositoryTestCase {
 
+    public static MinioFixtureTestContainer minioFixtureTestContainer = new MinioFixtureTestContainer(true);
+
+    @ClassRule
+    public static TestRule testRule = (base, description) -> {
```

Comment:
Why are we doing this rather than just annotating `minioFixtureTestContainer`?

Reply:
cleaned that up